### PR TITLE
Expose chart

### PR DESCRIPTION
**Pull request by peterjcoffman**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: cmp/chart/Chart.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ Chart.propTypes = {
 class LocalModel {
     @bindable aspectRatio;
     chartRef = createObservableRef();
-    chart = null;
+    _chart = null;
```
Comment (lbwexler) [inline cmp/chart/Chart.js]: Why are we duping this state?  Let's just rely on the Model's version of it.

File: cmp/chart/ChartModel.js [hunk 2]
```diff
@@ -16,6 +16,10 @@ export class ChartModel {
 
     @bindable.ref highchartsConfig = {};
     @bindable.ref series = [];
+    /**
+     * The HighCharts chart instance currently being displayed.
```
Comment (lbwexler) [inline cmp/chart/ChartModel.js]: Need the warnings here about:  not mutating this, use for reading state only, use the setHighchartsConfig() if you want to modify the underlying chart, etc.etc.

Comment (lbwexler) [reply cmp/chart/ChartModel.js]: also, we might want to call this `highchart` rather than hchart.

**Comment by lbwexler** [comment]
Ready to merge and just end the saga behind this little ticket.  

Good opportunity to improve our documentation on how you DO modify one of these charts.

**Comment by febbraiod** [comment]
Yay!

**Pull request merged**